contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.3.0--3.4.0.sql

```diff
@@ -2994,6 +2994,31 @@ $BODY$
 LANGUAGE plpgsql IMMUTABLE PARALLEL UNSAFE;
 GRANT EXECUTE ON FUNCTION sys.FORMAT(IN anyelement, IN NVARCHAR, IN VARCHAR) TO PUBLIC;
 
+-- Role member functions
+CREATE OR REPLACE FUNCTION sys.is_rolemember_internal(
```

Comment:
how we reached to the conclusion that database name cannot be sync between workers?

Reply:
Yeah, worker will use the same role as the leader. So, we should be able to pushdown the rolename to parallel worker and make it parallel safe.